src/components/pagination/index.md.njk

```diff
@@ -0,0 +1,118 @@
+---
+title: Pagination
+description: Help users navigate collections of numbered pages like search results
+section: Components
+aliases:
+backlog_issue_id: 77
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+Help users navigate a series of pages like search results or guides.
+
+{{ example({group: "components", item: "pagination", example: "default", html: true, nunjucks: true }) }}
+
+## When to use this component
+
+Use this component when you need to present content, such as guidance or search results, across multiple pages.
+
+You might do this when:
+- showing all the content on a single page makes the page take too long to load
+- users may benefit from bookmarking or linking to specific pages of the content
+- most users will only need the content on the first pages
+
+## When not to use this component
+
+Only break up content into pages if it improves the performance or usability of your service.
+
+Avoid using the 'infinite scroll' technique to automatically load content when the user approaches the bottom of the page. This causes problems for keyboard users.
+
+## How it works
+
+Add the pagination component after the content on each page that you have broken up.
+
+[View an example of the component in a standard GOV.UK page template](in-page/index.html).
+
+### For pages in a sequence
+
+For pages the user would usually navigate page by page, only show the previous and next links. Stack them  vertically, like this:
+
+{{ example({group: "components", item: "pagination", example: "labels", html: true, nunjucks: true }) }}
+
+#### Add link labels to describe pages
+
+You can use the link labels to give more context.
+
+{{ example({group: "components", item: "pagination", example: "labels-2", html: true, nunjucks: true }) }}
+
+### For pages in a list
+
+For pages where users might need to jump back and forth, add numbered page links, as well as the previous and next links, like this:
+
+{{ example({group: "components", item: "pagination", example: "default", html: true, nunjucks: true, titleSuffix: "second" }) }}
+
+Show the page number in the title of the page so the user can see it in browser tabs and bookmarks. For example 'Search results (page 1 of 4)'.
+
+Do not show pagination if there's only one page of content.
+
+Redirect users to the first page if they enter a URL of a page that no longer exists.
+
+#### Previous and next links
+
+Previous and next links are labelled 'Previous page' and 'Next page'. The word
+'page' can be visually hidden to conserve horizontal space.
+
+{{ example({group: "components", item: "pagination", example: "default", html: true, nunjucks: true, titleSuffix: "third" }) }}
+
+#### Customise link titles
+
+You can use custom text for previous and next links, if they're more appropiate and help add context.
```

Comment:
```suggestion
You can use custom text for previous and next links, if they're more appropriate and help add context.
```
Probably me being slow again, but it took me a couple of looks to realise that "custom text" matched up with 'Previous team' and 'Next team' in the example. Wondering if it might also be difficult for someone using a screen reader.

This might be overkill, but suggesting it in case it helps:

> You can use custom text (for example, 'Previous team') for previous and next links, if they’re more appropriate and help add context.